hr_employee_passport/models/hr_employee_passport.py

```diff
@@ -0,0 +1,79 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 ACSONE SA/NV
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class HrEmployeePassport(models.Model):
+
+    _name = 'hr.employee.passport'
+    _inherit = 'mail.thread'
+    _description = 'Employee Passport'
+    _order = 'end_date desc'
+
+    name = fields.Char(
```

Comment:
Indeed, but maybe ```passport_id``` should be a computed field which get the active passport number. What do you think?

Reply:
I have no issues, but it will give you some headaches to add them in a post_init _hook method, not to loose data, because your model  HrEmployeePassport contains required fields that you don't have it in Odoo.